app/controllers/concerns/with_organization.rb

```diff
@@ -6,4 +6,8 @@ def set_organization!
   def visit_organization!
     current_user.visit!(Organization.current)
   end
-end
\ No newline at end of file
+
+  def implicit_central?
+    !(request.first_subdomain == 'central') && Organization.central?
```

Comment:
PS: here there is the `request.first_subdomain` code: 

https://github.com/mumuki/mumukit-core/blob/master/lib/mumukit/core/request.rb